services/gitea/4.1.1/defaults/cm.yaml

```diff
@@ -9,11 +9,22 @@ data:
     ---
     image:
       rootless: true
+    ingress:
+      enabled: true
+      annotations:
+        kubernetes.io/ingress.class: kommander-traefik
+        traefik.ingress.kubernetes.io/router.middlewares: kommander-stripprefixes@kubernetescrd
+        traefik.ingress.kubernetes.io/router.tls: "true"
+      hosts:
+        - paths:
+          - path: /dkp/kommander/git
+            pathType: ImplementationSpecific
     gitea:
       admin:
         existingSecret: ${adminCredentialsSecret}
       config:
         server:
+          ROOT_URL: https://localhost/dkp/kommander/git
```

Comment:
Won't this need to be changed to use the ingress hostname?

Reply:
The only reason I'm setting this here is that the Helm chart won't install with it not being set if the ingress has no hostname set: https://gitea.com/gitea/helm-chart/src/branch/master/templates/gitea/config.yaml#L58-L68

Since we obviously can only know the hostname at runtime, I'm just setting `ROOT_URL` to a hard-coded value here. For our purposes of simply serving a Git repository over HTTPS this `ROOT_URL` doesn't matter.

Eventually we may want to adapt installation to set the actual hostname but that doesn't need to happen right now.